package.json

```diff
@@ -3,11 +3,17 @@
   "version": "10.7.1",
   "description": "Format whitespace in a SQL query to make it more readable",
   "license": "MIT",
-  "main": "lib/index.js",
+  "main": "dist/sql-formatter.min.js",
+  "module": "lib/index.js",
   "types": "lib/src/index.d.ts",
+  "exports": {
+    ".": "./lib/index.js"
+  },
```

Comment:
I wonder if it's really worth it to specify both `module` and `exports` fields. The `module` field was a proposed way of specifying exported module, but was never made official. Later the `exports` field was adopted in its place. Support for `exports` was added 2019 in NodeJS 12, which is no more actively supported.

I think `main` should serve as a good-enough fallback for those using some old NodeJS or some old bundler.

Reply:
you're correct in that module is not an official spec for npm packaging purposes but it is used unofficially by webpack for their new experimental ESM export config